scripts/bin/populate-activitypub-db

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
```

Comment:
Not a blocker - Kudos on persevering with raw bash here, the script is easy / clear to follow and grok 👌

Thoughts on using https://github.com/google/zx for stuff like this (not suggesting you change anything here 👍)?

Reply:
Oooh I remember seeing this before, still not tried it! I'm happy to give it a go!